_data/powered_by.yml

```diff
@@ -1,6 +1,4 @@
 - title: Who Uses Infer?
   items:
-  - name: Marks and Spencer
-    url: https://www.marksandspencer.com/
-  - name: Sky
-    url: https://www.sky.com
```

Comment:
You accidentally deleted the other users, please put them back and update your PR.